fixups for SD clock, DMAC and reset

xakep-amatop opened the pull request:
Add reset of MMC DMAC in case of DMA errors and to SDHC reset function.

    Fix description of 'rcar_mmc_init_start_clk' function.

    There isn't any sense to wait when RCAR_MMC_INFO2_CBSY flag is cleared,
    because reset function can be called for case when MMC controller stucks
    due to this flag.

    SD clock divider is set incorrectly for case when it is needed to supply
    from SDϕ without division.

lorc reviewed (commented):
Please remove "fixup" prefix in the commit message.  If you want to create proper git fixup, please use correct syntax: "fixup! <name of the original commit to be fixed>".

s/diveder/divider

s/ration/ratio

Also, I didn't understood commit message at all. Could you please make more clear description?

xakep-amatop committed:
fixup! driver: sdhc: add support of Renesas MMC driver

SD clock divider is set incorrectly for case when it is needed to supply
from SDϕ without division.

Signed-off-by: Mykola Kvach <mykola_kvach@epam.com>
Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
Acked-by: Dmytro Firsov <dmytro_firsov@epam.com>

xakep-amatop committed:
fixup! driver: sdhc: add support of Renesas MMC driver

There isn't any sense to wait when RCAR_MMC_INFO2_CBSY flag is cleared,
because reset function can be called for case when MMC controller stucks
due to this flag.

Signed-off-by: Mykola Kvach <mykola_kvach@epam.com>
Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
Acked-by: Dmytro Firsov <dmytro_firsov@epam.com>

xakep-amatop committed:
fixup! drivers: rcar_mmc: use clock set API for configuring SD/MMC rate

Fix description of 'rcar_mmc_init_start_clk' function.

Signed-off-by: Mykola Kvach <mykola_kvach@epam.com>
Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
Acked-by: Dmytro Firsov <dmytro_firsov@epam.com>

xakep-amatop committed:
fixup! drivers: rcar_mmc: add support of MMC DMA without Seq

Add reset of MMC DMAC in case of DMA errors and to SDHC reset function.

Signed-off-by: Mykola Kvach <mykola_kvach@epam.com>
Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
Acked-by: Dmytro Firsov <dmytro_firsov@epam.com>

lorc commented:
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

@firscity, are you okay with those patches being fixups?

firscity reviewed (approved):
`Acked-by: Dmytro Firsov <dmytro_firsov@epam.com>`

@lorc, now we are assembling `zephyr-v3.3.0-xt` branch, where we can squash them. So I think it is ok for `aos-wip` branch
